package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "react-trello",
+  "version": "1.20.4",
```

Comment:
The library uses semantic versioning, so version is not hardcoded in the package.json and will be generated automatically on build